ihatemoney/forms.py

```diff
@@ -174,13 +174,19 @@ class BillForm(FlaskForm):
     submit = SubmitField(_("Submit"))
     submit2 = SubmitField(_("Submit and add a new one"))
 
+    def __init__(self, bill_type=BillTypes.loan.value, *args, **kwargs):
+        super(BillForm, self).__init__(*args, **kwargs)
+        self.bill_type = bill_type
```

Comment:
To ease the logic, we might want to use strings for the types, and select the correct type here. It will allow to pass a string rather than a python class to the form.

Reply:
This still applies.